benchmark/test_blas_perf.py

```diff
@@ -47,12 +50,33 @@ def set_more_shapes(self):
     def get_tflops(self, op, *args, **kwargs):
         """This method is currently not really implemented and serves as a placeholder.
         A proper implementation will be developed in the future."""
-        from torch.utils.flop_counter import FlopCounterMode
-
         fn = lambda: op(*args, **kwargs)
-        with FlopCounterMode(display=False) as flop_counter:
-            fn()
-        tflops = flop_counter.get_total_flops()
+        #time(s)
+        latency = triton.testing.do_bench(
```

Comment:
Since latency is already handled in performance_util.py, it would be more appropriate to calculate only total_flops here and then compute tflops in the base Benchmark class (using the latency result). The function name get_tflops may have caused some confusion, and we plan to rename it to clarify this.

Reply:
Ok, I've noticed the problem, but I've calculated the time again because of the function name. I'll revise it next